src/Storefront/Resources/views/storefront/page/checkout/checkout-item.html.twig

```diff
@@ -107,14 +107,16 @@
                                                     {% endif %}
                                                 {% endblock %}
 
-                                                {% if lineItem.type == 'product' and lineItem.price.referencePrice is not null %}
-                                                    {% set price = lineItem.price %}
```

Comment:
This may break, if price is used inside below inside another block. If you put it outside the new block, it should keep working.

Reply:
Yeah, you are right. Variables inside twig blocks are lost on leaving the block. Good catch. I'll amend that.